android/src/main/java/com/tailscale/ipn/App.kt

```diff
@@ -92,6 +94,12 @@ class App : UninitializedApp(), libtailscale.AppContext {
         getString(R.string.taildrop_file_transfers),
         getString(R.string.notifications_delivered_when_a_file_is_received_using_taildrop),
         NotificationManagerCompat.IMPORTANCE_DEFAULT)
```

Comment:
This whole file could use a reformat... The indentation is all over the place....

Reply:
It actually looks nice in Android Studio (ktfmt is taking care of this). There's something off with the rendering on GitHub.